[2023-06-11T02:17:22.901+0200] {processor.py:156} INFO - Started process (PID=47307) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:17:22.903+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:17:22.908+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:17:22.906+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:17:23.969+0200] {logging_mixin.py:117} INFO - Writing here:  /home/choco/airflow/datalake/raw/Google/Datacsv/20230611/
[2023-06-11T02:17:23.978+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:17:23.977+0200] {dagbag.py:330} ERROR - Failed to import: /home/choco/airflow/dags/DAG2.py
Traceback (most recent call last):
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/choco/airflow/dags/DAG2.py", line 7, in <module>
    from lib.fetch_data_from_youtube import fetch_data_from_youtube
ImportError: cannot import name 'fetch_data_from_youtube' from 'lib.fetch_data_from_youtube' (/home/choco/airflow/dags/lib/fetch_data_from_youtube.py)
[2023-06-11T02:17:23.979+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:17:24.001+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 1.105 seconds
[2023-06-11T02:17:59.871+0200] {processor.py:156} INFO - Started process (PID=47561) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:17:59.872+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:17:59.873+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:17:59.873+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:18:00.614+0200] {logging_mixin.py:117} INFO - Writing here:  /home/choco/airflow/datalake/raw/Google/Datacsv/20230611/
[2023-06-11T02:18:00.619+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:18:00.619+0200] {dagbag.py:330} ERROR - Failed to import: /home/choco/airflow/dags/DAG2.py
Traceback (most recent call last):
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/choco/airflow/dags/DAG2.py", line 7, in <module>
    from lib.fetch_data_from_youtube import fetch_data_from_youtube
ImportError: cannot import name 'fetch_data_from_youtube' from 'lib.fetch_data_from_youtube' (/home/choco/airflow/dags/lib/fetch_data_from_youtube.py)
[2023-06-11T02:18:00.620+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:18:00.635+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.766 seconds
[2023-06-11T02:18:23.465+0200] {processor.py:156} INFO - Started process (PID=47591) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:18:23.467+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:18:23.468+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:18:23.468+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:18:41.654+0200] {processor.py:156} INFO - Started process (PID=47650) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:18:41.655+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:18:41.655+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:18:41.655+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:18:42.408+0200] {logging_mixin.py:117} INFO - Writing here:  /home/choco/airflow/datalake/raw/Google/Datacsv/20230611/
[2023-06-11T02:18:42.424+0200] {processor.py:768} INFO - DAG(s) dict_keys(['DAG2']) retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:18:42.437+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:18:42.437+0200] {dag.py:2573} INFO - Sync 1 DAGs
[2023-06-11T02:18:42.566+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:18:42.565+0200] {dag.py:3324} INFO - Setting next_dagrun for DAG2 to 2023-06-10T00:00:00+00:00, run_after=2023-06-11T00:00:00+00:00
[2023-06-11T02:18:42.592+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.940 seconds
[2023-06-11T02:19:26.751+0200] {processor.py:156} INFO - Started process (PID=47742) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:19:26.762+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:19:26.763+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:19:26.763+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:19:27.584+0200] {logging_mixin.py:117} INFO - Writing here:  /home/choco/airflow/datalake/raw/Google/Datacsv/20230611/
[2023-06-11T02:19:27.594+0200] {processor.py:768} INFO - DAG(s) dict_keys(['DAG2']) retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:19:27.697+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:19:27.697+0200] {dag.py:2573} INFO - Sync 1 DAGs
[2023-06-11T02:19:27.711+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:19:27.711+0200] {dag.py:3324} INFO - Setting next_dagrun for DAG2 to 2023-06-11T00:00:00+00:00, run_after=2023-06-12T00:00:00+00:00
[2023-06-11T02:19:27.729+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.980 seconds
[2023-06-11T02:19:36.966+0200] {processor.py:156} INFO - Started process (PID=47751) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:19:36.967+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:19:36.967+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:19:36.967+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:19:37.622+0200] {logging_mixin.py:117} INFO - Writing here:  /home/choco/airflow/datalake/raw/Google/Datacsv/20230611/
[2023-06-11T02:19:37.633+0200] {processor.py:768} INFO - DAG(s) dict_keys(['DAG2']) retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:19:37.738+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:19:37.738+0200] {dag.py:2573} INFO - Sync 1 DAGs
[2023-06-11T02:19:37.754+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:19:37.754+0200] {dag.py:3324} INFO - Setting next_dagrun for DAG2 to 2023-06-11T00:00:00+00:00, run_after=2023-06-12T00:00:00+00:00
[2023-06-11T02:19:37.773+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.808 seconds
[2023-06-11T02:19:49.815+0200] {processor.py:156} INFO - Started process (PID=47775) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:19:49.826+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:19:49.826+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:19:49.826+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:19:50.506+0200] {logging_mixin.py:117} INFO - Writing here:  /home/choco/airflow/datalake/raw/Google/Datacsv/20230611/
[2023-06-11T02:19:50.518+0200] {processor.py:768} INFO - DAG(s) dict_keys(['DAG2']) retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:19:50.623+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:19:50.622+0200] {dag.py:2573} INFO - Sync 1 DAGs
[2023-06-11T02:19:50.640+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:19:50.640+0200] {dag.py:3324} INFO - Setting next_dagrun for DAG2 to 2023-06-11T00:00:00+00:00, run_after=2023-06-12T00:00:00+00:00
[2023-06-11T02:19:50.660+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.846 seconds
[2023-06-11T02:21:52.589+0200] {processor.py:156} INFO - Started process (PID=47872) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:21:52.591+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:21:52.592+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:21:52.592+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:21:53.315+0200] {logging_mixin.py:117} INFO - Writing here:  /home/choco/airflow/datalake/raw/Google/Datacsv/20230611/
[2023-06-11T02:21:53.329+0200] {processor.py:768} INFO - DAG(s) dict_keys(['DAG2']) retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:21:53.339+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:21:53.339+0200] {dag.py:2573} INFO - Sync 1 DAGs
[2023-06-11T02:21:53.477+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:21:53.477+0200] {dag.py:3324} INFO - Setting next_dagrun for DAG2 to 2023-06-11T00:00:00+00:00, run_after=2023-06-12T00:00:00+00:00
[2023-06-11T02:21:53.498+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.910 seconds
[2023-06-11T02:22:29.802+0200] {processor.py:156} INFO - Started process (PID=47925) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:22:29.813+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:22:29.814+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:22:29.814+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:22:31.151+0200] {logging_mixin.py:117} INFO - Writing here:  /home/choco/airflow/datalake/raw/Google/Datacsv/20230611/
[2023-06-11T02:22:31.164+0200] {processor.py:768} INFO - DAG(s) dict_keys(['DAG2']) retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:22:31.294+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:22:31.294+0200] {dag.py:2573} INFO - Sync 1 DAGs
[2023-06-11T02:22:31.311+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:22:31.311+0200] {dag.py:3324} INFO - Setting next_dagrun for DAG2 to 2023-06-11T00:00:00+00:00, run_after=2023-06-12T00:00:00+00:00
[2023-06-11T02:22:31.330+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 1.531 seconds
[2023-06-11T02:22:37.211+0200] {processor.py:156} INFO - Started process (PID=47931) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:22:37.222+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:22:37.222+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:22:37.222+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:22:37.963+0200] {logging_mixin.py:117} INFO - Writing here:  /home/choco/airflow/datalake/raw/Google/Datacsv/20230611/
[2023-06-11T02:22:37.974+0200] {processor.py:768} INFO - DAG(s) dict_keys(['DAG2']) retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:22:38.093+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:22:38.093+0200] {dag.py:2573} INFO - Sync 1 DAGs
[2023-06-11T02:22:38.108+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:22:38.107+0200] {dag.py:3324} INFO - Setting next_dagrun for DAG2 to 2023-06-11T00:00:00+00:00, run_after=2023-06-12T00:00:00+00:00
[2023-06-11T02:22:38.124+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.915 seconds
[2023-06-11T02:22:48.739+0200] {processor.py:156} INFO - Started process (PID=47941) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:22:48.749+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:22:48.750+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:22:48.750+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:22:49.769+0200] {logging_mixin.py:117} INFO - Writing here:  /home/choco/airflow/datalake/raw/Google/Datacsv/20230611/
[2023-06-11T02:22:49.780+0200] {processor.py:768} INFO - DAG(s) dict_keys(['DAG2']) retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:22:49.906+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:22:49.906+0200] {dag.py:2573} INFO - Sync 1 DAGs
[2023-06-11T02:22:49.923+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:22:49.923+0200] {dag.py:3324} INFO - Setting next_dagrun for DAG2 to 2023-06-11T00:00:00+00:00, run_after=2023-06-12T00:00:00+00:00
[2023-06-11T02:22:49.944+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 1.207 seconds
[2023-06-11T02:23:05.384+0200] {processor.py:156} INFO - Started process (PID=47957) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:05.395+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:23:05.396+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:23:05.396+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:06.181+0200] {logging_mixin.py:117} INFO - Writing here:  /home/choco/airflow/datalake/raw/Google/Datacsv/20230611/
[2023-06-11T02:23:06.192+0200] {processor.py:768} INFO - DAG(s) dict_keys(['DAG2']) retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:06.307+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:23:06.306+0200] {dag.py:2573} INFO - Sync 1 DAGs
[2023-06-11T02:23:06.321+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:23:06.321+0200] {dag.py:3324} INFO - Setting next_dagrun for DAG2 to 2023-06-11T00:00:00+00:00, run_after=2023-06-12T00:00:00+00:00
[2023-06-11T02:23:06.338+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.956 seconds
[2023-06-11T02:23:21.824+0200] {processor.py:156} INFO - Started process (PID=47974) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:21.835+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:23:21.835+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:23:21.835+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:22.664+0200] {logging_mixin.py:117} INFO - Writing here:  /home/choco/airflow/datalake/raw/Google/Datacsv/20230611/
[2023-06-11T02:23:22.680+0200] {processor.py:768} INFO - DAG(s) dict_keys(['DAG2']) retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:22.786+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:23:22.786+0200] {dag.py:2573} INFO - Sync 1 DAGs
[2023-06-11T02:23:22.801+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:23:22.801+0200] {dag.py:3324} INFO - Setting next_dagrun for DAG2 to 2023-06-11T00:00:00+00:00, run_after=2023-06-12T00:00:00+00:00
[2023-06-11T02:23:22.819+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.996 seconds
[2023-06-11T02:23:33.031+0200] {processor.py:156} INFO - Started process (PID=47985) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:33.032+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:23:33.032+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:23:33.032+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:33.702+0200] {logging_mixin.py:117} INFO - Writing here:  /home/choco/airflow/datalake/raw/Google/Datacsv/20230611/
[2023-06-11T02:23:33.714+0200] {processor.py:768} INFO - DAG(s) dict_keys(['DAG2']) retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:33.826+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:23:33.826+0200] {dag.py:2573} INFO - Sync 1 DAGs
[2023-06-11T02:23:33.855+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:23:33.855+0200] {dag.py:3324} INFO - Setting next_dagrun for DAG2 to 2023-06-11T00:00:00+00:00, run_after=2023-06-12T00:00:00+00:00
[2023-06-11T02:23:33.873+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.844 seconds
[2023-06-11T02:23:47.408+0200] {processor.py:156} INFO - Started process (PID=47999) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:47.409+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:23:47.410+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:23:47.410+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:48.106+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:23:48.105+0200] {dagbag.py:330} ERROR - Failed to import: /home/choco/airflow/dags/DAG2.py
Traceback (most recent call last):
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/choco/airflow/dags/DAG2.py", line 6, in <module>
    from lib.fetch_data_from_google import fetch_data_from_google
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 57, in <module>
    fetch_data_from_google(keyword)
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 7, in fetch_data_from_google
    trends = visualize_related_topics_from_google(keyword)
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 26, in visualize_related_topics_from_google
    pytrend.build_payload(kw_list=[keyword], timeframe=f"{start_date_rising} {end_date}")
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 189, in build_payload
    self._tokens()
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 195, in _tokens
    widget_dicts = self._get_data(
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 159, in _get_data
    raise exceptions.TooManyRequestsError.from_response(response)
pytrends.exceptions.TooManyRequestsError: The request failed: Google returned a response with code 429
[2023-06-11T02:23:48.107+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:48.133+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.726 seconds
[2023-06-11T02:23:59.410+0200] {processor.py:156} INFO - Started process (PID=48012) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:59.422+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:23:59.422+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:23:59.422+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:59.791+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:23:59.790+0200] {dagbag.py:330} ERROR - Failed to import: /home/choco/airflow/dags/DAG2.py
Traceback (most recent call last):
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/choco/airflow/dags/DAG2.py", line 6, in <module>
    from lib.fetch_data_from_google import fetch_data_from_google
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 57, in <module>
    fetch_data_from_google(keyword)
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 7, in fetch_data_from_google
    trends = visualize_related_topics_from_google(keyword)
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 20, in visualize_related_topics_from_google
    pytrend.build_payload(kw_list=[keyword], timeframe=f"{start_date_top} {end_date}")
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 189, in build_payload
    self._tokens()
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 195, in _tokens
    widget_dicts = self._get_data(
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 159, in _get_data
    raise exceptions.TooManyRequestsError.from_response(response)
pytrends.exceptions.TooManyRequestsError: The request failed: Google returned a response with code 429
[2023-06-11T02:23:59.793+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:23:59.808+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.399 seconds
[2023-06-11T02:24:12.268+0200] {processor.py:156} INFO - Started process (PID=48024) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:24:12.269+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:24:12.269+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:24:12.269+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:24:12.639+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:24:12.639+0200] {dagbag.py:330} ERROR - Failed to import: /home/choco/airflow/dags/DAG2.py
Traceback (most recent call last):
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/choco/airflow/dags/DAG2.py", line 6, in <module>
    from lib.fetch_data_from_google import fetch_data_from_google
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 57, in <module>
    fetch_data_from_google(keyword)
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 7, in fetch_data_from_google
    trends = visualize_related_topics_from_google(keyword)
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 20, in visualize_related_topics_from_google
    pytrend.build_payload(kw_list=[keyword], timeframe=f"{start_date_top} {end_date}")
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 189, in build_payload
    self._tokens()
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 195, in _tokens
    widget_dicts = self._get_data(
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 159, in _get_data
    raise exceptions.TooManyRequestsError.from_response(response)
pytrends.exceptions.TooManyRequestsError: The request failed: Google returned a response with code 429
[2023-06-11T02:24:12.641+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:24:12.665+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.400 seconds
[2023-06-11T02:24:24.275+0200] {processor.py:156} INFO - Started process (PID=48036) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:24:24.276+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:24:24.277+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:24:24.277+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:24:24.637+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:24:24.636+0200] {dagbag.py:330} ERROR - Failed to import: /home/choco/airflow/dags/DAG2.py
Traceback (most recent call last):
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/choco/airflow/dags/DAG2.py", line 6, in <module>
    from lib.fetch_data_from_google import fetch_data_from_google
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 57, in <module>
    fetch_data_from_google(keyword)
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 7, in fetch_data_from_google
    trends = visualize_related_topics_from_google(keyword)
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 20, in visualize_related_topics_from_google
    pytrend.build_payload(kw_list=[keyword], timeframe=f"{start_date_top} {end_date}")
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 189, in build_payload
    self._tokens()
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 195, in _tokens
    widget_dicts = self._get_data(
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 159, in _get_data
    raise exceptions.TooManyRequestsError.from_response(response)
pytrends.exceptions.TooManyRequestsError: The request failed: Google returned a response with code 429
[2023-06-11T02:24:24.640+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:24:24.659+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.385 seconds
[2023-06-11T02:24:38.048+0200] {processor.py:156} INFO - Started process (PID=48049) to work on /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:24:38.050+0200] {processor.py:758} INFO - Processing file /home/choco/airflow/dags/DAG2.py for tasks to queue
[2023-06-11T02:24:38.051+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:24:38.051+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:24:38.433+0200] {logging_mixin.py:117} INFO - [2023-06-11T02:24:38.432+0200] {dagbag.py:330} ERROR - Failed to import: /home/choco/airflow/dags/DAG2.py
Traceback (most recent call last):
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/choco/airflow/dags/DAG2.py", line 6, in <module>
    from lib.fetch_data_from_google import fetch_data_from_google
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 57, in <module>
    fetch_data_from_google(keyword)
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 7, in fetch_data_from_google
    trends = visualize_related_topics_from_google(keyword)
  File "/home/choco/airflow/dags/lib/fetch_data_from_google.py", line 20, in visualize_related_topics_from_google
    pytrend.build_payload(kw_list=[keyword], timeframe=f"{start_date_top} {end_date}")
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 189, in build_payload
    self._tokens()
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 195, in _tokens
    widget_dicts = self._get_data(
  File "/home/choco/airflow_venv/lib/python3.10/site-packages/pytrends/request.py", line 159, in _get_data
    raise exceptions.TooManyRequestsError.from_response(response)
pytrends.exceptions.TooManyRequestsError: The request failed: Google returned a response with code 429
[2023-06-11T02:24:38.434+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/choco/airflow/dags/DAG2.py
[2023-06-11T02:24:38.459+0200] {processor.py:178} INFO - Processing /home/choco/airflow/dags/DAG2.py took 0.414 seconds
